open-ce/inputs.py

```diff
@@ -108,11 +108,11 @@ class Argument(Enum):
                                         help='CUDA version to build for '
                                              ', such as "10.2" or "11.0".'))
 
-    DOCKER_BUILD = (lambda parser: parser.add_argument(
-                                        '--docker_build',
+    CONTAINER_BUILD = (lambda parser: parser.add_argument(
+                                        '--container_build',
```

Comment:
Lets also keep `--docker_build` as an option here.

Reply:
Instead of creating a CONTAINER_BUILD value and DOCKER_BUILD value, you can just have the CONTAINER_BUILD one. When you call add_argument you can then do: 
`parser.add_argument('--container_build', '--docker_build', `
This will make it so passing either `--container_build` or `--docker_build` will have the same behavior.

Reply:
yes sure. Thanks @bnemanich.